### DISCUSSION TITLE
接入集群 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1070] [Announcements]
# 接入集群 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/cluster/join/

**Comment by liyuerich** [comment]
在弹框内输入污点的键值信息，值可以为空 - 目前不支持值为空

**Reply by Michelle951** [reply]
fixed, thanks
